src/openforms/submissions/tests/test_email_verification.py

```diff
@@ -190,3 +190,121 @@ def test_custom_email_subject_and_body(self):
             """,
             message_html,
         )
+
+
+class VerifyEmailTests(SubmissionsMixin, APITestCase):
```

Comment:
Should we add a test for multiple codes?For example if the user asks for a verification code more than once in the same submission?Or maybe I don't remember right, is that possible anyway?

Reply:
it is possible, I will add a test case :)